src/Workspaces/Core/Portable/Workspace/Solution/Checksum_Factory.cs

```diff
@@ -14,41 +15,44 @@ namespace Microsoft.CodeAnalysis
     // all these are just helper methods
     internal partial class Checksum
     {
+        private static readonly ObjectPool<IncrementalHash> s_incrementalHashPool =
+            new ObjectPool<IncrementalHash>(() => IncrementalHash.CreateHash(HashAlgorithmName.SHA1), size: 20);
```

Comment:
why 20?

Reply:
no particular reason. other pool uses 20 so, it uses 20 as well.